chromium_src/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc

```diff
@@ -366,11 +368,15 @@ void BraveAddAboutStrings(content::WebUIDataSource* html_source,
   html_source->AddString("aboutProductLicense", license);
 }
 
-void BraveAddSocialBlockingLoadTimeData(content::WebUIDataSource* html_source,
-                                        Profile* profile) {
+void BraveAddExtensionSettingsLoadTimeData(
```

Comment:
the old name didn't make sense, but neither does this one and I don't think this needs to be a method anyway, just put them directly in `BraveAddLocalizedStrings`

Reply:
resolved!